front/src/pages/settings/SettingsWorkspace.tsx

```diff
@@ -0,0 +1,3 @@
+export function SettingsWorkspace() {
+  return <div>Workspace</div>;
```

Comment:
Let's try to always build something functional and start with small to grow big. Otherwise, this produces non releasable code.
Here I would start with a Settings page with only Profile containing read-only email + firstname + lastname. In another PR we can add Logout.

Then later we can work on workspace page

Reply:
Ok, as long as this is shippable, I'm happy!